Sources/Hub/Hub.swift

```diff
@@ -161,8 +167,10 @@ public class LanguageModelConfigurationFromHub {
         }
     }
 
-    func loadConfig(modelName: String, hfToken: String? = nil) async throws -> Configurations {
-        let hubApi = HubApi(hfToken: hfToken)
+    func loadConfig(
+        modelName: String,
+        hubApi: HubApi = HubApi()
```

Comment:
```suggestion
        hubApi: HubApi = HubApi.shared
```

Same as before, unless you think it's better to instantiate